packages/notifi-frontend-client/lib/NotifiFrontendClient.ts

```diff
@@ -1,12 +1,29 @@
 import { NotifiService } from '@notifi-network/notifi-graphql';
-import { UserFragmentFragment } from 'notifi-graphql/lib/gql/generated';
 
+import {
+  BeginLogInByTransactionResult,
+  CompleteLogInByTransactionInput,
+  CompleteLogInByTransactionMutation,
+  UserFragmentFragment,
+  WalletBlockchain,
+} from '../../notifi-graphql/lib/gql/generated';
 import type { NotifiFrontendConfiguration } from './configuration/NotifiFrontendConfiguration';
 import type { NotifiStorage } from './storage';
 import { notNullOrEmpty } from './utils/notNullOrEmpty';
 
 export type SignMessageFunction = (message: Uint8Array) => Promise<Uint8Array>;
 
+type BeginLoginProps = {
+  walletBlockchain: WalletBlockchain;
+  walletPublicKey: string;
+};
```

Comment:
Omit<BeginLogInByTransactionInput, 'dappAddress'>`

Reply:
Sweet, +1 to utility types: typescriptlang.org/docs/handbook/utility-types.html#omittype-keys